MMM-TeslaFi.js

```diff
@@ -254,6 +254,36 @@ Module.register("MMM-TeslaFi", {
             `;
           }
           break;
+
+        case "state":
+          table += `
+					<tr>
+						<td class="icon"><span class="zmdi zmdi-hc-fw ${
+              t.carState === "Sentry"
+                ? "sentry zmdi-dot-circle"
+                : "zmdi-parking"
+            }"></span></td>
+						<td class="field">State</td>
+						<td class="value">${t.carState}</td>
+					</tr>
+				`;
+          if (t.carState === "Driving") {
```

Comment:
I think it'd be nice to split the `speed` and `heading` out of here and in to their own individual fields - what do you think?

Reply:
Fully agree, on it!